operate/webapp/src/main/java/io/camunda/operate/webapp/opensearch/reader/OpensearchListenerReader.java

```diff
@@ -64,15 +66,18 @@ public List<ListenerDto> getListenerExecutions(
                     request.getSorting().getSortBy(),
                     getSortOrder(request.getSorting().getSortOrder())));
 
-    final List<ListenerDto> listenerDtos =
-        richOpenSearchClient.doc().search(searchRequest, JobEntity.class).hits().hits().stream()
+    final SearchResult<JobEntity> searchResult =
+        richOpenSearchClient.doc().search(searchRequest, JobEntity.class);
+    final int totalHitCount = (int) searchResult.hits().total().value();
+    final List<Object> listenerDtos =
```

Comment:
You can declare this as `List<ListenerDto>`. Then you don't have to use a raw list in the `ListenerResponseDto` constructor.